actioncable/test/subscription_adapter/postgresql_test.rb

```diff
@@ -12,7 +12,7 @@ def setup
     if Dir.exist?(ar_tests)
       require File.join(ar_tests, "config")
       require File.join(ar_tests, "support/config")
-      local_config = ARTest.config["arunit"]
+      local_config = ARTest.config["connections"]["postgresql"]["arunit"]
```

Comment:
It can be extracted into a separate PR, 'cause it does nothing with the bug under consideration.